src/main/kotlin/com/vk/admstorm/startup/ChangeSshBackendStartup.kt

```diff
@@ -10,44 +10,50 @@ import com.vk.admstorm.notifications.AdmNotification
 object ChangeSshBackendStartup {
     private val LOG = logger<ChangeSshBackendStartup>()
 
-    private const val SSH_NOTIFICATOR_OPTION = "sshLegacyChanges"
+    private const val LEGACY_SSH_NOTIFICATOR_OPTION = "sshLegacyChanges"
     private const val SSH_CONFIG_BACKEND = "ssh.config.backend"
+    private const val SSH_NOTIFICATOR_OPTION = "sshOpenSSHChanges"
 
     private fun changeSshConfiguration(configValue: SshConnectionConfigService.Kind) {
         AdvancedSettings.setEnum(SSH_CONFIG_BACKEND, configValue)
     }
 
     fun changeConfigurationProcess(project: Project) {
+        //TODO: Remove it after 2 month
+        if (PropertiesComponent.getInstance(project).getValue(LEGACY_SSH_NOTIFICATOR_OPTION) != null) {
+            PropertiesComponent.getInstance(project).unsetValue(LEGACY_SSH_NOTIFICATOR_OPTION)
+        }
+
         val dntShow = PropertiesComponent.getInstance(project).getBoolean(SSH_NOTIFICATOR_OPTION)
         if (dntShow) {
-            LOG.info("LEGACY SSH option was enabled")
+            LOG.info("OPENSSH option was enabled")
             return
         }
 
         val sshSettingValue = AdvancedSettings.getEnum(SSH_CONFIG_BACKEND, SshConnectionConfigService.Kind::class.java)
-        if (sshSettingValue == SshConnectionConfigService.Kind.LEGACY) {
-            PropertiesComponent.getInstance(project).setValue(SSH_NOTIFICATOR_OPTION, true)
-            LOG.info("LEGACY SSH option was enabled before")
+        if (sshSettingValue == SshConnectionConfigService.Kind.OPENSSH) {
+            PropertiesComponent.getInstance(project).setValue(LEGACY_SSH_NOTIFICATOR_OPTION, true)
```

Comment:
Why `LEGACY_SSH_NOTIFICATOR_OPTION` and not `SSH_NOTIFICATOR_OPTION`?

Reply:
Thank you, fixed